src/overlays/actors/ovl_En_Invadepoh_Demo/z_en_invadepoh_demo.c

```diff
@@ -54,115 +124,689 @@ static InitChainEntry D_80C1AA88[] = {
+    }
+}
+
+void EnInvadepohDemo_Alien_Draw(EnInvadepohDemo* this, PlayState* play) {
+    s32 pad[2];
+    Mtx* mtx;
+
+    OPEN_DISPS(play->state.gfxCtx);
+
+    Gfx_SetupDL25_Xlu(play->state.gfxCtx);
+    Gfx_SetupDL25_Opa(play->state.gfxCtx);
+    Matrix_Push();
+    AnimatedMat_Draw(play, Lib_SegmentedToVirtual(&gAlienEmptyTexAnim));
+    Scene_SetRenderModeXlu(play, 0, 1);
+    gDPSetEnvColor(POLY_OPA_DISP++, 0, 0, 0, 255);
+    POLY_OPA_DISP =
+        SkelAnime_DrawFlex(play, this->skelAnime.skeleton, this->skelAnime.jointTable, this->skelAnime.dListCount, NULL,
+                           EnInvadepohDemo_Alien_PostLimbDraw, &this->actor, POLY_OPA_DISP);
+
+    // In EnInvadepoh, the eye beam alpha is controlled by an instance variable, and there is an if-block that checks
+    // to see if this variable is not zero. In this actor, the eye beam alpha is hardcoded, but this block is still
+    // necessary to prevent reordering; this is likely a result of copy-pasting from EnInvadepoh.
+    if (true) {
+        Gfx* gfx;
+
+        AnimatedMat_Draw(play, Lib_SegmentedToVirtual(&gAlienEyeBeamTexAnim));
+
+        OPEN_DISPS(play->state.gfxCtx);
+
+        gfx = POLY_XLU_DISP;
+        gDPPipeSync(gfx++);
+        gDPSetPrimColor(gfx++, 0, 0xFF, 240, 180, 100, 60);
+        gDPSetEnvColor(gfx++, 255, 255, 255, 150);
+        Matrix_Mult(&sAlienLeftEyeBeamMtxF, MTXMODE_NEW);
+
+        mtx = Matrix_NewMtx(play->state.gfxCtx);
+
+        if (mtx != NULL) {
+            gSPMatrix(gfx++, mtx, G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            gSPDisplayList(gfx++, gAlienEyeBeamDL);
+            Matrix_Mult(&sAlienRightEyeBeamMtxF, MTXMODE_NEW);
+
+            mtx = Matrix_NewMtx(play->state.gfxCtx);
+
+            if (mtx != NULL) {
+                gSPMatrix(gfx++, mtx, G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+                gSPDisplayList(gfx++, gAlienEyeBeamDL);
+                POLY_XLU_DISP = gfx;
+            }
+        }
+
+        CLOSE_DISPS(play->state.gfxCtx);
+    }
+
+    CLOSE_DISPS(play->state.gfxCtx);
+
+    // In EnInvadepoh, the code to draw the light flash checks an instance variable which is set to true if the actor
+    // should draw. In this actor, an equivalent variable is checked in EnInvadepohDemo_Draw, so it isn't checked here.
+    // However, this block is still necessary to prevent reordering; this is likely a result of copy-pasting from
+    // EnInvadepoh. Similarly, this function requires a lot of extra stack space for unused variables; the names and
+    // types of these variables are inferred from EnInvadepoh's version of this code.
+    if (true) {
+        Gfx* gfx;
+        Vec3f glowOffset;
+        Vec3f glowPos; // unused, inferred from EnInvadepoh
+        s32 glowAlpha; // unused, inferred from EnInvadepoh
+
+        OPEN_DISPS(play->state.gfxCtx);
+
+        gfx = POLY_XLU_DISP;
+        gfx = Gfx_SetupDL20_NoCD(gfx);
+        gSPSetOtherMode(gfx++, G_SETOTHERMODE_H, 4, 4, 0x00000080);
```

Comment:
```suggestion
        gDPSetDither(gfx++, G_CD_NOISE);
```
There are 2 other uses one in `EnOsk_Draw` and one in `func_80B32F04` which could alos be changed, but I'll leave that up to you if you want.

If you do end up fixing them as well, could you also remove the `if(1)`'s in `EnOsk_Draw` by doing 
```c
    gfx = POLY_XLU_DISP;
    gfx = Gfx_SetupDL20_NoCD(gfx);
```
like what is done in `func_80B32F04`

Reply:
Just ended up fixing them all because why not? Might as well do it right.